src/Core/Components/SortableList/FluentSortableList.razor.cs

```diff
@@ -0,0 +1,84 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+using Microsoft.JSInterop;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+public partial class FluentSortableList<TItem> : FluentComponentBase
+{
+    private const string JAVASCRIPT_FILE = "./_content/Microsoft.FluentUI.AspNetCore.Components/Components/SortableList/FluentSortableList.razor.js";
+
+    [Inject]
+    private IJSRuntime JSRuntime { get; set; } = default!;
+
+    [Parameter]
+    public RenderFragment<TItem>? SortableItemTemplate { get; set; }
+
+    [Parameter, AllowNull]
+    public List<TItem> Items { get; set; }
+
+    [Parameter]
+    public EventCallback<(int oldIndex, int newIndex)> OnUpdate { get; set; }
+
+    [Parameter]
+    public EventCallback<(int oldIndex, int newIndex)> OnRemove { get; set; }
+
+    [Parameter]
```

Comment:
Could you add Comments to write the documentation ?

Reply:
Is on the list